libpius/signer.py

```diff
@@ -645,6 +648,9 @@ def sign_uid(self, key, index, level):
     while True:
       line = gpg.stdout.readline()
       debug('Got %s' % line)
+      if 'MISSING_PASSPHRASE' in line:
```

Comment:
From my testing both messages happen in either version.  The difference seems to be that `MISSING_PASSPHRASE` comes up when you cancel pinentry without entering a passphrase, where as `BAD_PASSPHRASE` means you got it wrong.  I found that if I cancelled pinentry without a phase phase pius would continue like normal.

So `MISSING_PASSPHRASE` not gpg2 specific (and `BAD_PASSPHRASE` is not gpg1 specific.)  I should have commented to point that out :)